tools/src/main/python/dlpx/virtualization/_internal/plugin_importer.py

```diff
@@ -199,6 +199,14 @@ def _get_manifest(queue, src_dir, module, entry_point, plugin_type, validate):
         #
         return
 
+    manifest = get_manifest_helper(src_dir, module, entry_point,
+                                   module_content, plugin_type,
+                                   validate, queue)
+    queue.put({'manifest': manifest})
+
+
+def get_manifest_helper(src_dir, module, entry_point, module_content,
```

Comment:
Can you add docstrings as to why this method is needed - this helps in the actual prod code and also mostly for unit tests on Windows due to difference in how multiprocessing module is implemented.